charts/argo-workflows/README.md

```diff
@@ -260,7 +260,7 @@ Fields to note:
 | server.GKEmanagedCertificate.domains | list | `["argoworkflows.example.com"]` | Domains for the Google Managed Certificate |
 | server.GKEmanagedCertificate.enabled | bool | `false` | Enable ManagedCertificate custom resource for Google Kubernetes Engine. |
 | server.affinity | object | `{}` | Assign custom [affinity] rules |
-| server.authMode | string | `""` | Auth Mode is available from `server` , `client` or `sso`. If you chose `sso` , please configure `.Values.server.sso` as well. |
+| server.authMode | list | `[]` | Auth Mode is available from `server` , `client` or `sso`. If you chose `sso` , please configure `.Values.server.sso` as well. |
```

Comment:
Maybe change it to `A list of supported authentication modes. Available values are server, client, or sso. If you provide sso, please configure .Values.server.sso as well.`. That makes it clearer that multiple values can be used.

Reply:
Thank you for your review. I fixed in https://github.com/argoproj/argo-helm/pull/2336/commits/3366c7c98737398940e18467b4806f27b14f6a3e 🙋